docs/zh/latest/how-to-build.md

```diff
@@ -94,6 +95,23 @@ sudo yum install -y https://repos.apiseven.com/packages/centos/apache-apisix-rep
   make install
   ```
 
+## 步骤2：安装 ETCD
+
+通过 RPM 或者 Docker 安装 Apache APISIX 时，需要执行此步骤。
+
+你可以通过 Docker 或者二进制等方式安装 ETCD。以下命令通过二进制方式安装 ETCD。
+
+```shell
+ETCD_VERSION='3.4.13'
```

Comment:
Maybe add a `$` prefix is clearer in the multiple lines 😄

Reply:
done